app/screens/settings/Settings.js

```diff
@@ -173,7 +178,23 @@ class Settings extends Component<Props, State> {
                 upperPartRight={<Link onClick={() => {}} text="CREATE" isDisabled />}
                 rowName="Create a new wallet"
               />
-              <SettingRow upperPartLeft={version} upperPartRight={<Link onClick={() => {}} text="CHECK FOR UPDATES" isDisabled />} rowName="Spacemesh Wallet Version" />
+              <SettingRow
+                upperPartLeft={version}
+                upperPartRight={[
+                  isUpdateDownloading === true && (
```

Comment:
simplify instead of "isUpdateDownloading === true" and "isUpdateDownloading === false"